.github/workflows/atd_moped_build_stage.yml

```diff
@@ -87,7 +87,6 @@ jobs:
           echo "ACTION/BRANCH_NAME: ${BRANCH_NAME}"
           echo "MOPED_PRNUM: ${MOPED_PRNUM}"
           source $(pwd)/.github/workflows/aws-heroku-helper.sh
-          clone_hasura_repo
```

Comment:
clone_hasura_repo was invoked here but also within build_database. I don't think it needs to be invoked twice, but let me know if theres some reason why it should happen in both places.